### PR TITLE
t/m/interfaces-network-manager: use different channel depending on system

### PR DESCRIPTION
**Pull request by alfonsosanchezbeato**
Install network-manager from different channels depending on system
under test. Due to tight integration with UC, each UC version requires
a different NM snap.

### CODE REVIEW

File: tests/main/interfaces-network-manager/task.yaml [hunk 1]
```diff
@@ -20,7 +20,15 @@ systems:
     - ubuntu-core-20-64
 
 prepare: |
-    echo "Give a network-manager snap is installed"
+    echo "Given a network-manager snap is installed"
+    if [ "$SPREAD_SYSTEM" = ubuntu-core-16-32 ] || [ "$SPREAD_SYSTEM" = ubuntu-core-16-64 ]; then
+        snap install --channel=latest network-manager
+    elif [ "$SPREAD_SYSTEM" = ubuntu-core-18-32 ] || [ "$SPREAD_SYSTEM" = ubuntu-core-18-64 ]; then
```
Comment (anonymouse64) [inline tests/main/interfaces-network-manager/task.yaml:26]: same for `os.query is-core18`

**Review by anonymouse64 — COMMENTED** [review]
slight preference to use os.query for this, otherwise lgtm

**Review by stolowski — APPROVED** [review]
Thanks, looks good but I agree with @anonymouse64 suggestion re os.query.

**Review by sergiocazzolato — APPROVED** [review]
Thanks for the fix, I made a quick fix to make it mergeable

**Review by anonymouse64 — APPROVED** [review]
thanks

**Comment by alfonsosanchezbeato** [comment]
> Thanks for the fix, I made a quick fix to make it mergeable

Thanks for that! (sorry, got too deep into other things)

**Pull request merged**